src/py/mat3ra/made/material.py

```diff
@@ -86,8 +86,16 @@ def set_new_lattice_vectors(
         if original_is_in_crystal_units:
             self.to_crystal()
 
+    def set_new_lattice_vectors_from_vectors_array(self, lattice_vectors: List[List[float]]) -> None:
```

Comment:
this name -> set_lattice_vectors_from_array

set_new_lattice_vectors -> set_lattice_vectors